aries-site/src/examples/components/pagination/PaginationExample.js

```diff
@@ -1,4 +1,6 @@
 import React from 'react';
 import { Pagination } from 'grommet';
 
-export const PaginationExample = () => <Pagination numberItems={200} />;
+export const PaginationExample = () => (
+  <Pagination stepOptions summary numberItems={200} />
```

Comment:
can we add the border and pad to this so it's reflective of how we want teams to implement? @britt6612

Reply:
@jcfilben are you comfortable with that?

Reply:
done

Reply:
Yeah I'm good with that

Reply:
sounds good @taysea you good to merge?